### PR TITLE
assignments outgoing

### PR DESCRIPTION
**Pull request by chriku**

### CODE REVIEW

File: sync/src/main/kotlin/gropius/sync/AbstractSync.kt [hunk 1]
```diff
@@ -882,10 +906,87 @@ abstract class AbstractSync(
      * @param timeline Timeline of the issue
      * @param imsProject IMS project to sync
      * @param issueInfo Issue to sync
+     * @param label Label to sync
```
Comment (nk-coding) [inline sync/src/main/kotlin/gropius/sync/AbstractSync.kt:909]: ?

File: sync/src/main/kotlin/gropius/sync/AbstractSync.kt [hunk 2]
```diff
@@ -882,10 +906,87 @@ abstract class AbstractSync(
...
+     * @param issueInfo Issue to sync
+     */
+    private suspend fun syncOutgoingSingleAssignment(
+        relevantTimeline: List<TimelineItem>,
+        imsProject: IMSProject,
+        issueInfo: IssueConversionInformation,
+        assignment: Assignment,
+        virtualIDs: Map<TimelineItem, String>
+    ) {
+        var labelIsSynced = false
```
Comment (nk-coding) [inline sync/src/main/kotlin/gropius/sync/AbstractSync.kt:948]: wrong name

File: sync/src/main/kotlin/gropius/sync/AbstractSync.kt [hunk 3]
```diff
@@ -882,10 +906,87 @@ abstract class AbstractSync(
...
+        val finalBlock = findFinalTypeBlock(relevantTimeline)
+        for (item in finalBlock) {
+            val relevantEvent = collectedSyncInfo.timelineItemConversionInformationService.findByImsProjectAndGropiusId(
+                imsProject.rawId!!, item.rawId ?: virtualIDs[item]!!
+            )
+            if (relevantEvent?.githubId != null) {
+                labelIsSynced = true
+            }
+        }
+        if (!labelIsSynced) {
```
Comment (nk-coding) [inline sync/src/main/kotlin/gropius/sync/AbstractSync.kt:958]: a bit long

**Review by nk-coding — CHANGES_REQUESTED** [review]

File: sync/src/main/kotlin/gropius/sync/AbstractSync.kt (hunk 3 above)
Comment (chriku) [reply sync/src/main/kotlin/gropius/sync/AbstractSync.kt]: b7bc985fcf236b4b2c81866b22cefb9d15d0f7f3

**Pull request merged**